src/Kamino.ts

```diff
@@ -1393,6 +1401,47 @@ export class Kamino {
     return { a: balances.tokenAAmounts.div(sharesIssued), b: balances.tokenBAmounts.div(sharesIssued) };
   };
 
+  /**
+   * Get all scope prices feeds (token A, token B and rewards) for the specified strategy
+   * @param strategy
+   * @param collateralInfos the collateral infos array
+   */
+  getAllScopePriceFeedsForStrategy = (strategy: WhirlpoolStrategy, collateralInfos: CollateralInfo[]): Address[] => {
```

Comment:
i think it would be cleaner to return a struct so it is clear which feed is for which token
```
struct StrategyTokenFeeds {
  tokenAFeed: Address,
  tokenBFeed: Address,
  rewardsFeeds: []Address,
  kaminoRewardsFeeds: []Address
}
```

Reply:
also would be nicer to pass this as param to various functions so we don't rely on the order in the list